docs/modules/demos/pages/end-to-end-security.adoc

```diff
@@ -5,20 +5,26 @@
 This is a demo to showcase what can be done with Open Policy Agent around authorization in the Stackable Data Platform.
 It covers the following aspects of security:
 
-* Column- and row-level filtering
-* OIDC support across the board
-* Kerberos on Kubernetes
-* Keycloak and flexible group lookup
-* Open Policy Agent for the utmost flexibility in building access rules
 
-across this tech stack:
 
-* Trino
-* Superset
-* HDFS
-
-
-Install this demo on an existing Kubernetes cluster:
+This demo will:
+
+* Install the Stackable operators
+* Spin up the following data products
+** *Trino*: A fast distributed SQL query engine for big data analytics that helps you explore your data universe. This demo uses it to enable SQL access to the data.
+** *Spark*: A multi-language engine for executing data engineering, data science, and machine learning. This demo uses it to batch-process data from S3 by training and scoring an unsupervised anomaly detection model and writing the results into a Trino table. Spark uses an isolation forest algorithm from the scikit-learn machine learning library in this demo.
```

Comment:
Is this correct? Does this do the scikit-learn stuff? Or is this a copy & paste error from the anomaly demo?

Reply:
You are right, thanks! c11b151b16a50e712741cec3e7d5bc5b976dc9ce